bolt-expo/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "bolt-expo-starter",
+  "main": "expo-router/entry",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "expo start --clear",
+    "build:web": "expo export --platform web",
+    "test": "jest --watchAll",
+    "lint": "expo lint"
+  },
+  "jest": {
+    "preset": "jest-expo"
+  },
```

Comment:
do we really need to include jest? no files reference it and there's no config

Reply:
Yea, we can prolly just remove it.

Reply:
Let's keep it super barebone 👍

Reply:
Done.

Reply:
in that case, maybe we can remove all of this: 

```
    "@types/react-test-renderer": "^18.3.0",
    "jest": "^29.2.1",
    "jest-expo": "~52.0.3",
    "react-test-renderer": "18.3.1",
```